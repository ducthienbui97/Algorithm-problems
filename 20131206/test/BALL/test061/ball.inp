8 864105 205084 63124 235557
412463 457588 834147 491829 413497 510970 350826 541351 582543 741986 215626 830096 218463 685227 703490 178387 716153 323688 999719 544644 223061 282739 518177 543732 43237 106097 580269 797509 323238 73617 840553 307857 824644 642857 396528 869442 315675 799727 131589 767100 46389 788240 876515 823936 50695 239248 34528 82042 462521 501949 662566 215465 887496 894411 959373 297037 501330 824588 653882 113833 129242 418309 10400 91374 
231628 410435 132366 569676 175452 823484 894303 635372 230103 684889 259138 707673 701176 393192 419713 864253 991351 473999 883964 428731 231542 596928 266563 846804 633650 680946 826910 170155 224641 885535 732327 186804 879250 176806 184729 694543 659026 370292 787133 99830 554429 548251 905008 716174 335589 655623 968066 345883 769160 337162 25931 395175 782287 354037 759070 126144 269111 399307 587430 848975 543848 16688 194951 874814 
617317 970076 361394 812746 396693 953308 739558 978950 995238 464487 652445 258620 20623 392177 239897 299633 632050 573022 560595 840588 445160 857248 431521 875895 36048 272285 79284 90303 10241 490155 412855 682018 578921 674758 590859 175171 747153 857260 648442 786320 318447 985858 961978 949388 647189 867461 51320 549283 950995 995894 1822 185003 895423 453641 464189 124282 342801 37619 671040 847709 943715 35697 236765 444248 
706286 702830 858078 606714 307209 791964 35412 710863 421862 952460 847759 872075 436179 191130 415100 369970 33511 312192 396554 47063 921045 721466 638080 262630 633311 631840 621869 968973 29017 414856 969481 996210 720061 380514 71892 973155 23902 830421 409596 203749 594199 611040 351734 409780 262100 718287 941562 619635 381199 149896 861271 378405 877721 421426 142207 823318 805723 669063 90961 133702 607413 24771 228879 286060 881615 387936 2251 443288 396081 172945 335827 225947 655948 737481 260717 526372 912059 18524 588995 646934 370589 329760 783645 909423 453353 718567 361860 962939 894743 416820 928838 365083 402295 87558 913093 375853 536896 49354 856702 940013 673281 288731 119453 489702 198315 769275 58700 20948 470567 858416 456231 484086 933002 35860 925797 298204 933155 136660 468134 114675 947928 11200 976150 752158 429755 481022 533817 340774 634643 549557 222203 799570 825646 638577 375933 894353 949333 51601 813475 302542 485740 932548 679672 623019 680582 366109 971778 483753 867298 594660 75961 234290 819450 74084 520899 120640 958404 496030 284205 343518 704506 480871 66974 579734 182838 344220 861499 10469 666085 154931 185258 490383 755911 946970 895465 370226 677110 823504 758335 368572 381539 358473 96385 894669 149021 345174 802814 436588 739752 68056 435114 807722 164014 772071 497502 152885 415643 470872 605719 491128 331174 461989 127416 558649 429298 137400 674984 100871 667720 3878 154569 719174 91326 364947 339371 884318 155838 216625 963344 117054 609675 461309 103386 998287 748254 875569 155519 405930 178574 202362 316298 750264 721232 335501 13939 74095 367120 640582 462069 549739 993096 887614 452067 577989 841979 344636 66738 853732 222036 909694 534286 214923 374407 309714 992008 433401 753909 395487 815826 932417 285247 44155 768785 910355 9706 423326 26690 680449 946496 958540 829813 799393 589095 234006 170957 166337 347027 185238 317730 505180 652254 241367 163835 210726 939783 23511 389824 771176 568700 783182 968685 442227 941504 937009 735062 357405 549533 846789 458387 86441 727727 869603 713324 895193 393500 199117 679533 139880 274715 298738 324172 835546 536122 365503 322513 1559 783420 721788 159212 58962 785250 738556 603906 163289 669790 783442 203087 375515 361755 877735 968455 934506 215698 476613 622621 692702 436359 884550 799838 979133 418871 306187 998152 322238 499319 392293 16443 527972 975552 995093 107578 685395 669318 553560 267575 544191 736116 109858 499954 446606 965595 281688 623116 903066 918615 763968 813917 566247 746773 208143 396337 291784 453143 113590 558771 781720 96732 585806 336148 656806 364687 918244 653821 642385 601553 479802 191603 543599 750661 432910 916468 584029 30341 135028 115470 167574 254156 28032 660582 154436 8542 270952 889284 388892 963408 584720 306708 349909 679058 915383 141595 851881 593737 989375 304531 335882 12032 2662 809534 35851 166044 251712 353361 850988 256115 53545 404281 257705 99219 532402 227061 497338 398035 522953 128270 228979 861363 46138 929299 793724 180900 343395 919006 797112 671837 664883 755937 864817 462853 543131 280107 52543 924746 6542 275107 333612 228507 50671 183446 7858 297087 399650 577713 927822 49936 704113 644825 669618 630766 401201 94312 215621 833228 89866 562314 297914 559223 239156 355141 118980 136540 855227 69761 51357 731114 266631 373189 517506 292484 809258 90844 329320 158244 99735 280884 232354 570429 529587 925431 339207 170792 457710 685552 215876 595460 632253 544900 94570 99096 51316 108208 116767 