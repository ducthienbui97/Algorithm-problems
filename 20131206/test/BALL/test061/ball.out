8555977417661
